bitnami/nginx/README.md

```diff
@@ -66,6 +66,9 @@ The following tables lists the configurable parameters of the NGINX chart and th
 | `commonLabels`                          | Labels to add to all deployed objects                      | `{}`                                                    |
 | `commonAnnotations`                     | Annotations to add to all deployed objects                 | `{}`                                                    |
 | `extraDeploy`                           | Array of extra objects to deploy with the release          | `[]` (evaluated as a template)                          |
+| `pdb.enabled`                           | Enabled PodDisruptionBudget                                | `false`                                                 |
```

Comment:
```suggestion
| `pdb.create`                            | Creates PodDisruptionBudget                                | `false`                                                 |
```

Reply:
Its copied from the cassandra pdb, like suggested from @FraPazGal. I can change it if you want.